### PR TITLE
add tfenv switch to installing with tfenv

### PR DESCRIPTION
**Pull request by michalbagrowski**
```
bash-4.4# which tfenv
/usr/local/tfenv-2.2.2/bin/tfenv
bash-4.4# terraform version
wTerraform v0.12.30

Your version of Terraform is out of date! The latest version
is 1.1.2. You can update by downloading from https://www.terraform.io/downloads.html
bash-4.4# which terraform
/usr/local/tfenv-2.2.2/bin/terraform
bash-4.4# tfenv use 1.1.2
Switching default version to v1.1.2
Switching completed
bash-4.4# terraform version
Terraform v1.1.2
on linux_amd64
bash-4.4# which terraform
/usr/local/tfenv-2.2.2/bin/terraform
bash-4.4#
```

### CODE REVIEW

File: ansible/setup.yml [hunk 1]
```diff
@@ -140,12 +167,12 @@
...
-        domain: "{{ project_user }}"
-        limit_type: soft
-        limit_item: nofile
-        value: 24576
+#    - name: Increase ulimit
+#      pam_limits:
+#        domain: "{{ project_user }}"
+#        limit_type: soft
+#        limit_item: nofile
+#        value: 24576
```
Comment (venimus) [inline ansible/setup.yml:175]: i think this ulimit is still "required" by node

Comment (michalbagrowski) [reply ansible/setup.yml]: yeap, it was just ocal debugging 🗡️

**Review by mmpetarpeshev — APPROVED** [review]
LGTM

**Review by mmpetarpeshev — APPROVED** [review]
LGTM. Not sure what versions should be there  , but I believe they are tested.

**Review by venimus — DISMISSED** [review]
lgtm

**Review by venimus — APPROVED** [review]

**Review by venimus — DISMISSED** [review]

**Review by dincho — CHANGES_REQUESTED** [review]
Please use a concrete version (freeze, keep the sort order) in the requirements file, that's half of the point of it. `requests` and `urllib3`

File: ansible/setup.yml [hunk 2]
```diff
@@ -111,7 +111,7 @@
         - ansible
         - boto
         - boto3
-        - datadog==0.28.0
+        - datadog
```
Comment (dincho) [inline ansible/setup.yml:114]: Have you verified the new version works ? I bet it's frozen for a reason

Comment (michalbagrowski) [reply ansible/setup.yml:114]: what was reason, what should be checked?

Comment (venimus) [reply ansible/setup.yml:114]: i missed that change, i think there were some breaking changes that is why it was locked, should be checked if it works -> send properly data to datadog, may be investigate the changelog of the agent

Comment (venimus) [reply ansible/setup.yml:114]: probably leave that for a separate PR

Comment (michalbagrowski) [reply ansible/setup.yml]: @dincho @venimus  i have locked version back :D changing is not worth testing ;]

**Comment by michalbagrowski** [comment]
@dincho @venimus fixed all requested issues ;]

**Review by venimus — APPROVED** [review]

**Review by dincho — APPROVED** [review]

**Pull request merged**